### ISSUE TITLE
Upgrade rubocop.yml according to the warning

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch-ruby#66]
Currently, when we run
```bash
$ bundle exec rubocop lib/ spec/
```
```bash
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Layout/EmptyLinesAroundAttributeAccessor (0.83)
 - Layout/SpaceAroundMethodCallOperator (0.82)
 - Lint/DeprecatedOpenSSLConstant (0.84)
 - Lint/MixedRegexpCaptureTypes (0.85)
 - Lint/RaiseException (0.81)
 - Lint/StructNewOverride (0.81)
 - Style/ExponentialNotation (0.82)
 - Style/HashEachMethods (0.80)
 - Style/HashTransformKeys (0.80)
 - Style/HashTransformValues (0.80)
 - Style/RedundantFetchBlock (0.86)
 - Style/RedundantRegexpCharacterClass (0.85)
 - Style/RedundantRegexpEscape (0.85)
 - Style/SlicingWithRange (0.83)
For more information: https://docs.rubocop.org/rubocop/versioning.html
```

We should fix the warning with the more convenient configuration for our code base.

**Comment by Jeremy-Walton** [comment]
Created https://github.com/meilisearch/meilisearch-ruby/pull/97 to address this

**Comment by curquiza** [comment]
Closed with #97
